assets/javascripts/discourse/initializers/discourse-calendar.js

```diff
@@ -833,10 +833,11 @@ function initializeDiscourseCalendar(api) {
     );
     const timezoneButton = document.createElement("button");
 
+    timezoneButton.title = "Toggle timezone offset";
```

Comment:
This needs to be localizable, something like: 

```javascript
I18n.t("discourse_calendar.toggle_timezone_offset_title");
```

and then `toggle_timezone_offset_title` should be added to client.en.yml.